ios/Plugin/InAppReview.swift

```diff
@@ -4,7 +4,13 @@ import StoreKit
 
 @objc public class InAppReview: NSObject {
     @objc func requestReview(_ call: CAPPluginCall) {
-        SKStoreReviewController.requestReview()
+        if #available(iOS 14.0, *) {
+             if let windowScene = UIApplication.shared.connectedScenes.first(where: { $0.activationState == .foregroundActive }) as? UIWindowScene {
+                 SKStoreReviewController.requestReview(in: windowScene)
+             }
+         } else {
+             SKStoreReviewController.requestReview()
+         }
```

Comment:
I noticed a slowdown in my app and an error in the console. I had to make one change to ensure the request ran on the UI thread. 

```suggestion
        DispatchQueue.main.async {
            if #available(iOS 14.0, *) {
                if let windowScene = UIApplication.shared.connectedScenes.first(where: { $0.activationState == .foregroundActive }) as? UIWindowScene {
                    SKStoreReviewController.requestReview(in: windowScene)
                }
            } else {
                SKStoreReviewController.requestReview()
            }
```

Reply:
Yeah i could reproduce the issue. Thank you!